locales/pl/copyright-petition.properties

```diff
@@ -20,3 +20,9 @@ Petition_Optin=Tak, chcę otrzymywać wiadomości e-mail o kampaniach Mozilli.
 Petition_Privacy=Zgadzam się, aby Mozilla postępowała z moimi danymi zgodnie ze swoimi zasadami prywatności.
 Petition_Why_Ask=Dlaczego prosimy o te informacje?
 Petition_required=Wymagane pola
+Petition_We_care=Zależy nam na Twojej prywatności i chcemy pomóc Tobie dokonywać przemyślanych decyzji. Z tego powodu udostępniamy nasze zasady prywatności tak, aby można było je łatwo przeczytać. Do wysyłania wiadomości e-mail potrzebujemy adresu e-mail, imienia i kraju. Dzięki tej ostatniej informacji możemy wysyłać lokalne aktualności i wydarzenia najbardziej istotne dla Ciebie.
```

Comment:
I don't think the first quoted sentence is related to privacy policy, but rather the newsletters and such. I have changed the second sentence, though.